Add page "Comparison with other EDA tools" (#25)
LibrePCB/LibrePCB.github.io: content/compare/index.md

for details.

### LibrePCB vs. Eagle

- LibrePCB is free of charge - without limitations, even for commercial usage
- LibrePCB has an integrated library manager, thus it's much easier to install
  and update libraries
- LibrePCB library and project files are better suited for version control
- The schematic- and board editors of Eagle are currently more powerful
  (LibrePCB does not yet support hierarchical sheets, DRC, BOM export, 3D view
  and more)

### LibrePCB vs. KiCad

**Library management with LibrePCB is much easier:**

- Easy-to-use library manager to install and update libraries - no
  knowledge about Git and GitHub needed
- LibrePCB does not bundle (possibly outdated or low-quality) libraries together
  with the application - you have the full control over what libraries are used
- Only one library file format for all kinds of library elements - a library can
  contain symbols, footprints, components and in future also 3D models
- No broken references - you can rename symbols, footprints, pins, pads etc.
  without breaking other elements

> - LibrePCB organizes parts by category, thus it's easier and more intuitive to

Reviewer: ```suggestion
- LibrePCB organizes parts by hierarchical categories, thus it's easier and more intuitive to
```

  find the part you're looking for
- No broken projects caused by updated or removed libraries - LibrePCB projects
  are self-contained and completely independent of system libraries

> **Version control of LibrePCB files makes more fun:**

Reviewer: ```suggestion
**Version control of LibrePCB files is more fun:**
```

I don't think "make fun" is proper English :)

> - All files of LibrePCB projects and libraries are highly optimized for usage of

Reviewer: ```suggestion
- All files of LibrePCB projects and libraries are highly optimized for
```

  version control systems
- User-related settings are strictly separated from project-related settings so

>   you don't commit annoying, useless changes

Reviewer: ```suggestion
  you don't commit annoying, useless changes (like your zoom level or canvas position)
```

> - LibrePCB files are canonical - if you save a project without making relevant

Reviewer: ```suggestion
- LibrePCB files have a canonical file format - if you save a project without making relevant
```

Author: I reworded it to "LibrePCB has a canonical file format".

  changes, no files are modified at all

**Netlist synchronization is easier with LibrePCB:**

- Schematics and board work on the same netlist, so they are always in sync - no
  manual forward/backward annotation is needed
- You never have to worry about the pinout of components - LibrePCB stores all
  connections between symbol pins and footprint pads in the library

**BUT:**

Although LibrePCB has many cool advantages, KiCad is (currently) much more
powerful. It has an amazing amount of features which allow to design very
complex PCBs. LibrePCB is still a very young software and thus lacks many
features needed to design complex PCBs.

So, if you are looking for an intuitive EDA tool to quickly design a simple PCB,
you should give LibrePCB a try. But if you want to design complex PCBs, LibrePCB

> is probably not the tool you are looking for.

Reviewer: ```suggestion
is probably not (yet) the tool you are looking for.
```
